c10/xpu/XPUException.h

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+#include <c10/util/Exception.h>
+#include <sycl/sycl.hpp>
+
+namespace c10::xpu {
+
+static sycl::async_handler asyncHandler = [](sycl::exception_list el) {
+  for (auto& e : el) {
+    try {
+      std::rethrow_exception(e);
+    } catch (sycl::exception& e) {
+      TORCH_WARN(
+          "SYCL Exception: ",
+          e.what(),
+          "file = ",
+          __FILE__,
+          "line = ",
+          __LINE__);
```

Comment:
I guess the `__FILE__` and `__LINE__` here would be reported in the `XPUException.h` (not the source of the exception). Is it expected?

Reply:
Yes, they are useless information. removed.